### PR TITLE
chore: generated CHANGELOG for origin/release/v51.16.0

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Changelog for 51.16.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+- Added Java support for `SFDX: Start Function` ([PR #3269](https://github.com/forcedotcom/salesforcedx-vscode/pull/3269))
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Update default debug port ([PR #3272](https://github.com/forcedotcom/salesforcedx-vscode/pull/3272))
+
+- Use new cli for functions invoke @W-9282250@ ([PR #3270](https://github.com/forcedotcom/salesforcedx-vscode/pull/3270))
```
Comment (xyc) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]:
```suggestion
- Use new CLI for functions invoke @W-9282250@ ([PR #3270](https://github.com/forcedotcom/salesforcedx-vscode/pull/3270))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
+# 51.16.0 - June 3, 2021
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Add Java support for Functions debugging ([PR #3273](https://github.com/forcedotcom/salesforcedx-vscode/pull/3273))
```
Comment (xyc) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]:
```suggestion
- Added Java support for Functions debugging ([PR #3273](https://github.com/forcedotcom/salesforcedx-vscode/pull/3273))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+
+#### salesforcedx-vscode-core
+
+- Update default debug port ([PR #3272](https://github.com/forcedotcom/salesforcedx-vscode/pull/3272))
+
+- Use new cli for functions invoke @W-9282250@ ([PR #3270](https://github.com/forcedotcom/salesforcedx-vscode/pull/3270))
+
+- Throw error on remote component not found ([PR #3259](https://github.com/forcedotcom/salesforcedx-vscode/pull/3259))
+
+- Pull func dependencies by default + java/maven support @W-9282387@ ([PR #3262](https://github.com/forcedotcom/salesforcedx-vscode/pull/3262))
```
Comment (xyc) [inline packages/salesforcedx-vscode/CHANGELOG.md:21]:
```suggestion
- Pull function dependencies by default + Java/Maven support @W-9282387@ ([PR #3262](https://github.com/forcedotcom/salesforcedx-vscode/pull/3262))
```

File: packages/salesforcedx-vscode/CHANGELOG.md (hunk 2 above)
Comment (jag-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md:7]: We went over a similar comment couple of weeks ago and decided to use present tense.
I've committed the other changes though.

**Review by xyc — APPROVED** [review]

**Pull request merged**